build.gradle

```diff
@@ -100,17 +102,23 @@ gradlePlugin {
     }
 }
 
+def isProdPortal = !System.getProperty('gradle.portal.url')
 pluginBundle {
     website = 'https://github.com/gradle/android-cache-fix-gradle-plugin'
     vcsUrl = 'https://github.com/gradle/android-cache-fix-gradle-plugin'
 
     tags = ['android', 'cache', 'fix']
 
-    mavenCoordinates {
-        groupId = "gradle.plugin.org.gradle.android"
+    // The legacy groupId gradle.plugin.* is only allowed when the plugin
+    // has already been published
+    if (isProdPortal) {
```

Comment:
Why do we need this check? is it so we can publish to the dev portal with a different group id?

Reply:
exactly, did not want to mention the dev portal here since it's kinda external